schemas/com.snowplowanalytics.snowplow/media_player_event_error/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event tracked when the resource could not be loaded due to an error.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_error",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
```

Comment:
Error code / label?

Reply:
Added error code and description properties (similar to how they are tracked in the ecommerce events).